packages/ui-components/src/numeric-input/numeric-input.tsx

```diff
@@ -246,7 +246,7 @@ const NumericInput = React.forwardRef<HTMLInputElement, NumericInputProps>(
                 props.onChange?.(parsed, e);
             },
             // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Can we remove the disabled lint rule now?

Reply:
Right it's that rule again. Sure let's leave for now. For reference I believe it's requiring `props` because of `props.onChange`, in theory `props` reference could change without `onChange` changing and thus providing a different `this` reference to the function :D